.github/workflows/pr-preview.yml

```
@@ -25,6 +25,15 @@ jobs:
           workflow: ${{github.event.workflow_run.workflow_id}}
           run_id: ${{github.event.workflow_run.id}}
           if_no_artifact_found: ignore
+      - name: Download Test Coverage Artifact
```

```suggestion
      - name: Download Coverage Artifact
```

nit: to match "Upload Coverage Artifact" (or change the other one, I just think the step names should match)